[Docs] Documentation fix for the Examples Hub (#6109)
microsoft/MixedRealityToolkit-Unity: Documentation/README_ExampleHub.md

Example of loading multiple scenes.

## Running the scene ##

> The scene works on both Unity's game mode and the device. Run **MRTKExamplesHub** scene in the Unity editor and use MRTK's input simulation to enter and interact with the scene contents. To build and deploy, simply build **MRTKExamplesHub** scene with other scenes that are included in the Scene System's list. The inspector also makes it easy to add scenes to the Build Settings. In the Building Settings, make sure **MRTKExamplesHub** scene is on the top of the list.

Reviewer: ```suggestion
The scene works in both Unity's game mode and on device. Run the **MRTKExamplesHub** scene in the Unity editor and use MRTK's input simulation to interact with the scene contents. To build and deploy, simply build **MRTKExamplesHub** scene with other scenes that are included in the Scene System's list. The inspector also makes it easy to add scenes to the Build Settings. In the Building Settings, make sure **MRTKExamplesHub** scene is on the top of the list at index 0.
```

```
 
## Returning to the main menu scene ##

> To return to the main menu scene (MRTKExamplesHubMainMenu scene), you can use the exact same method. **ToggleFeaturesPanelExamplesHub.prefab** provides the 'Home' button which contains the **LoadContentScene** script. Use this prefab or provide your custom home button in each scene to allow the user to return to the main scene. You can put this **ToggleFeaturesPanelExamplesHub.prefab** in the **MRTKExamplesHub** scene to make it always visible since **MRTKExamplesHub** is shared container scene. Make sure to hide/deactivate **ToggleFeaturesPanel.prefab** in each example scene.

Reviewer: ```suggestion
To return to the main menu scene (MRTKExamplesHubMainMenu scene), you can use the same Scene System `LoadContent()` method. The **ToggleFeaturesPanelExamplesHub.prefab** provides the 'Home' button which contains the **LoadContentScene** script. Use this prefab or provide a custom home button in each scene to allow the user to return to the main scene. One can put the **ToggleFeaturesPanelExamplesHub.prefab** in the **MRTKExamplesHub** scene to make it always visible since **MRTKExamplesHub** is a shared container scene. Make sure to hide/deactivate **ToggleFeaturesPanel.prefab** in each example scene.
```
